src/client/components/EventOverview.less

```diff
@@ -58,18 +58,9 @@
 
```

Comment:
what's the difference between .less and .css?

Reply:
Less a css pre-processor. It does some cool stuff that css alone can't do; like mixins, variables, nested rules, and macros. It can generate pretty gnarly css if used recklessly (deeply nested selectors in particular can generate extremely verbose, overly specific css), but I find it makes some things easier when used carefully.